lemminx-maven/src/main/java/org/eclipse/lemminx/maven/DiagnosticRequest.java

```diff
@@ -57,4 +62,40 @@ public Range getRange() {
 				((DOMElement) node).getEndTagOpenOffset(), xmlDocument);
 	}
 
+	@Override
+	public int getOffset() {
+		return 0;
+	}
+
+	@Override
+	public Position getPosition() {
+		return null;
+	}
+
+	@Override
+	public DOMElement getParentElement() {
+		return this.node.getParentElement();
+	}
+
+	@Override
+	public String getCurrentTag() {
+		return this.node.getLocalName();
+	}
+
+	@Override
+	public String getCurrentAttributeName() {
+		return null;
+	}
+
+	@Override
+	public LineIndentInfo getLineIndentInfo() throws BadLocationException {
+		return null;
+	}
+
+	@Override
+	public <T> T getComponent(Class clazz) {
```

Comment:
@mickaelistria I think this patch is good, except I dislike the fact that I left some methods unimplemented. It should be ok for now, but calling these functions could lead to problems for DiagnosticRequest's.